src/current/v24.3/performance-recipes.md

```diff
@@ -297,6 +302,14 @@ A low percentage of live data can cause statements to scan more data ([MVCC valu
 
 Reduce the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) zone configuration of the table as much as possible.
 
+### KV DistSender batches being throttled (performance impact to larger clusters)
+
+If you see `distsender.batches.async.throttled` values that aren't zero (or aren't consistently near zero), experiment with increasing the [KV layer `DistSender`]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#distsender) and `Streamer` concurrency using the `kv.streamer.concurrency_limit` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}).  In v24.3, these default values were increased by 6x and 12x, respectively.  For versions prior to v24.3, increasing the value by 6x and 12x would be a good starting point.
+
+To validate a successful result, you can increase this value until you see no new throttled requests and no increase in tail latency (e.g. `p99.999`).
```

Comment:
It's not clear what "this value" refers to since these seem to be two values. Should it be "increase both values"? Should you always increase them together?

```suggestion
To validate a successful result, you can increase this value until you see no new throttled requests and no increase in tail latency (e.g., `p99.999`).
```